azure-pipelines.yml

```diff
@@ -108,6 +108,7 @@ stages:
         workingDirectory: cli
 
       - script: |
+          # I don't understand this. Isn't this a no-op if the second form isn't used: `npm link --help`?
```

Comment:
from "npm help link" ...


This is handy for installing your own stuff, so that you can work on it and test iteratively without having to continually rebuild.

Package linking is a two-step process.

First, npm link in a package folder will create a symlink in the global folder {prefix}/lib/node_modules/<package> that links to the package where the npm link command was executed. It will also link any bins in the package to {prefix}/bin/{name}. Note that npm link uses the global prefix (see npm prefix -g for its value).

Next, in some other location, npm link package-name will create a symbolic link from globally-installed package-name to node_modules/ of the current folder.

Note that package-name is taken from package.json, not from the directory name.

The package name can be optionally prefixed with a scope. See [scope](https://github.com/iTwin/pcf/pull/40/using-npm/scope.html). The scope must be preceded by an @-symbol and followed by a slash.

When creating tarballs for npm publish, the linked packages are "snapshotted" to their current state by resolving the symbolic links, if they are included in bundleDependencies.

Reply:
Right, I don't see where we're doing the second part, creating a symbolic link from the generated application's `node_modules` to the global `pcf` core. Instead we're `npm install ../../core`'ing. I think it's benign. I wish Zach were still here haha.

Reply:
Here's the change set where the only other reference to "Link" was added https://github.com/iTwin/pcf/commit/6788bfecef5836b2081435becd3b268484740fe6

I thought maybe that it once did the second part hence the comment, but, it looks like it always did the npm install.

Reply:
Okay I'm comfortable calling it a no-op then unless someone uses the globally installed `pcf`.